tests/test_logic.py

```diff
@@ -103,6 +103,8 @@ def test_render(template, var, expected_query, expected_params):
         (lambda: V('x').comma(V('y')), 'x, y'),
         (lambda: V('a').asc(), 'a ASC'),
         (lambda: V('a').desc(), 'a DESC'),
+        (lambda: V('a').nulls_first(), 'a NULLS FIRST'),
```

Comment:
can we just try `V('a').desc().nulls_first()` and check the precedence order makes sense.

Reply:
Sure - have updated